docs/source/partials/examples/reverse-city-search/_demo.html.erb

```diff
@@ -0,0 +1,40 @@
+<div>
+<p>When constructing a search UI using InstantSearch, you may want to filter your dataset to only display some of your products based on the distance to the end user. This is usually done using the aroundLatLngViaIP filter, or aroundLatLng filter if you have access to precise geolocation information. However geographical filters are hard to interpret when displayed in raw format, as noone really knows where the coordinates <span id="coords"><code>48.8566, 2.34287</code></span> are.</p>
+
```

Comment:
This bit is included in the demo part of the example because it updates to `#coords` element with the geolocation of the city that is found. 

I found this to be a sneaky way of illustrating the point that nobody understands where coordinates are just by reading them.